README.md

```diff
@@ -1 +1,250 @@
-## PERSONAL PROFOLIO PROJECT
\ No newline at end of file
+<a name="readme-top"></a>
+
+
+
+<div align="center">
+  
+  <img src="./images/logo.png" alt="logo" width="140"  height="auto" />
+  <br/>
+
+  <h3><b>PERSONAL PORTFOLIO PROJECT</b></h3>
+
+</div>
+
+<!-- TABLE OF CONTENTS -->
+
+# 📗 Table of Contents
+
+- [📖 About the Project](#about-project)
+  - [🛠 Built With](#built-with)
+    - [Tech Stack](#tech-stack)
+    - [Key Features](#key-features)
+  - [🚀 Live Demo](#live-demo)
+- [💻 Getting Started](#getting-started)
+  - [Setup](#setup)
+  - [Prerequisites](#prerequisites)
+  - [Install](#install)
+  - [Usage](#usage)
+  - [Run tests](#run-tests)
+  - [Deployment](#deployment)
+- [👥 Authors](#authors)
+- [🔭 Future Features](#future-features)
+- [🤝 Contributing](#contributing)
+- [⭐️ Show your support](#support)
+- [🙏 Acknowledgements](#acknowledgements)
+- [❓ FAQ (OPTIONAL)](#faq)
+- [📝 License](#license)
+
+
+<!-- PROJECT DESCRIPTION -->
+
+# 📖 [HELLO MICROVERSE PROJECT] <a name="about-project"></a>
+
+  **[Hello Microverse Project]** This is a personal portfolio project where I have implemented the main features of the project using images and backgrounds from Figma template, I have used  the  main concepts of flexbox layout to design a flexbox responsive layout structure acording to the methodology 'Mobil first'.
```

Comment:
- [ ] I believe you forgot to change the project name in the description as requested by the previous reviewer. Please make your readme file more descriptive in order to avoid any confusion for visitors. :ok_hand: 

```suggestion
# 📖 [PORTFOLIO PROJECT] <a name="about-project"></a>

  **[PORTFOLIO PROJECT]** This is a personal portfolio project where I have implemented the main features of the project using images and backgrounds from Figma template, I have used  the  main concepts of flexbox layout to design a flexbox responsive layout structure acording to the methodology 'Mobil first'.
```

Reply:
Fixed.